src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -327,6 +327,7 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 			// Type MakeGenericType (params Type[] typeArguments)
 			//
 			case IntrinsicId.Type_MakeGenericType: {
+					// We don't yet handle the very unlikely case where you can create a nullable type with typeof(Nullable<>).MakeGenericType(T)
```

Comment:
This is actually not that unlikely, but it's fine to not handle it right now.

Reply:
Updated the comment accordingly.